canonical-websites/www.ubuntu.com
# Fixed 500 error page

## Done

- Fixed 500 error page by removing nav and footer and use of {{ asset server url }}

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [500 page](http://0.0.0.0:8001/500)
- see that the page looks good

## Issue / Card

Fixes # 2591

## Screenshots
![image](https://user-images.githubusercontent.com/441217/35091175-842b425c-fc44-11e7-8908-61bbbad61f17.png)
